File: blook_app_flutter/lib/models/password_dto.dart
class PasswordDto {
  PasswordDto({
    required this.password,
    required this.passwordNew,
    required this.passwordNew2,
  });
  late final String password;
  late final String passwordNew;
  late final String passwordNew2;

  Map<String, dynamic> toJson() {
    final _data = <String, dynamic>{};
    _data['password'] = password;
    _data['passwordNew'] = passwordNew;
    _data['passwordNew2'] = passwordNew2;
    return _data;
  }
}

File: blook/src/main/java/com/salesianostriana/blook/dtos/PasswordDto.java
package com.salesianostriana.blook.dtos;

import com.salesianostriana.blook.validation.anotations.PasswordsMatch;
import lombok.*;
import javax.validation.constraints.NotBlank;
import javax.validation.constraints.Size;

@AllArgsConstructor
@NoArgsConstructor
@Getter @Setter
@Builder
@PasswordsMatch(passwordField = "passwordNew",
        verifyPasswordField = "passwordNew2",
        message = "{user.password.notmatch}")
public class PasswordDto {

    @NotBlank(message = "{user.password.blank}")
    private String password;
    @Size(min = 8, message = "{user.password.size}")
    @NotBlank(message = "{user.password.blank}")
    private String passwordNew;
    @Size(min = 8, message = "{user.password.size}")
    @NotBlank(message = "{user.password.blank}")
    private String passwordNew2;
}
